### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#26] [Announcements]
# about

开始练习写作的小白，我觉得相比写什么、怎么写、写得如何来说，更重要的是坚持写作。

https://tangly1024.com/about

**Comment by tangly1024** [comment]
测试一下

**Comment by 2Circle** [comment]
博主现在在哪里工作呢？ 我在厦门哈～

**Reply by tangly1024** [reply]
目前在福州哦～但是公司隶属于厦门😜

**Comment by SkysCrystal** [comment]
发现大佬网站没有被百度收录？

**Reply by tangly1024** [reply]
是啊，看来得找百度理论理论😂

**Comment by De4tsh** [comment]
大佬大佬，我是www.shtwo.top的”站长“，那个那个友链里的站名帮帮修改为SHTWO BLOG，现在这个感觉好二丫哈哈哈哈哈

**Reply by tangly1024** [reply]
哈哈哈哈哈，好哒好哒来啦

**Comment by songxurui** [comment]
博主好，我是你的粉丝！请问文章里的分割线怎么实现的？我notion里的分割线，渲染到博客是空白

**Reply by tangly1024** [reply]
Hi~粉丝你好，分割线是我自己添加的样式，每个H1标题都被渲染成底部带分割线的效果。

>  我notion里的分割线，渲染到博客是空白

这应该算是一个bug，notion的分割线在博客中默认被渲染成透明的空行，最新的版本中我将修复这个问题。

**Comment by wangyunzi** [comment]
大佬，你好我是一个今天才发现这样的新世界的人，就是想问一下大佬，我前面都按照你的步骤布置好了，但是在notion里面写文章和改变图标却在网站没有反应显示出来，想请问需要什么像hugo那样推送吗？非常感谢啦

**Reply by tangly1024** [reply]
Hi~ 你好😄，热烈欢迎你的加入🎉，看到你的网站和反馈。
首先可以确定的是：不需要像hugo这样的额外推送操作。无论是Github中的代码、还是Notion中的博文及站点描述会自动同步到你的网页。
针对你提的问题，我想可能需要具体检查一下：
 
1. > notion里面写文章 没有在网站反应显示
- 检查一下你的文章属性是否是符合标准的，其中最重要的是```status```必须为```publish```
2. > 图标改变没有反应出来 
- 具体是指哪个图标呢，如果正确修改了模板的图标是会实时显示为你的作者头像的。

另外，站点可能会有一会儿的缓存，一般需要刷新几次网页才能生效。
最后问题，如果无法解决的话可以[点此提交一个issue](https://github.com/tangly1024/NotionNext/issues/new?assignees=tangly1024&labels=deployment&template=deployment-error.md&title=)，描述一下你的详细操作过程，我帮你进行排查

**Comment by JensonMiao** [comment]
测试下

**Comment by Asterecho** [comment]
#  是否考虑国内笔记平台，担心notion会封

**Comment by CZZ12138** [comment]
用好久了，太感谢了

**Comment by uniech** [comment]
老哥，GitHub项目里的主题预览貌似失效了，链接进去没有对应的主题，也无法切换
